dependencies.props

```diff
@@ -171,11 +172,5 @@
       <PackageId>microsoft.xunit.runner.uwp</PackageId>
       <Version>$(AppXRunnerVersion)</Version>
     </DependencyBuildInfo>
-
-    <!-- project.json files to update -->
-    <ProjectJsonFiles Include="$(MSBuildThisFileDirectory)external\**\project.json" />
```

Comment:
Yep, it does. It will only update dependencies.props now, but the properties defined in there are used in the depproj files.